.github/workflows/github-actions-stale.yml

```diff
@@ -0,0 +1,25 @@
+name: Mark stale issues and pull requests
+on:
+  schedule:
+    - cron: '0 23 * * *' # once a day at 11pm UTC time zone
+jobs:
+  stale:
+    permissions:
+      issues: write # for commenting on an issue and editing labels
+      pull-requests: write # for commenting on a PR and editing labels
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          # timing
+          days-before-stale: 60 # 60 days of inactivity
+          days-before-close: 7 # 7 more days of inactivity
+          # labels to watch for, add, and remove
+          only-labels: 'pending info' # only mark issues/PRs as stale if they have this label
+          labels-to-remove-when-unstale: 'pending info' # remove label when unstale -- should be manually added back if information is insufficient
+          # automated messages to issue/PR authors
+          stale-issue-message: 'This issue is marked as stale because it has been open 60 days with no activity. Remove pending info & stale labels or comment if you want to address this issue otherwise this will be closed in 7 days.'
+          stale-pr-message: 'This pull request is marked as stale because it has been open 60 days with no activity. Remove pending info & stale labels or address review comments and update the pull request otherwise this will be closed in 7 days.'
+          close-issue-message: 'This issue has been closed due to inactivity and lack of information.If you still encounter this issue, please add the requested information and re-open.'
+          close-pr-message: 'This PR has been closed due to inactivity and lack of changes.If you would like to still work on this PR, please address the review comments and re-open.'
```

Comment:
Space after `.` is recommended in above two line.

Reply:
done